page01_index.php

```diff
@@ -21,7 +21,8 @@
 		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
 		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 		<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
-		
+		<!-- favicon -->
+		<link rel="icon" type="image/x-icon" href="images/favicon1.ico">
```

Comment:
added favicon: as well as images for each placeholder and graphics on the footer